defaults/defaults.go

```diff
@@ -58,10 +58,10 @@ const (
 	ClusterMeshApiserverImage             = "quay.io/cilium/clustermesh-apiserver"
 	ClusterMeshServiceName                = "clustermesh-apiserver"
 	ClusterMeshSecretName                 = "cilium-clustermesh" // Secret which contains the clustermesh configuration
-	ClusterMeshServerSecretName           = "clustermesh-apiserver-server-certs"
-	ClusterMeshAdminSecretName            = "clustermesh-apiserver-admin-certs"
-	ClusterMeshClientSecretName           = "clustermesh-apiserver-client-certs"
-	ClusterMeshExternalWorkloadSecretName = "clustermesh-apiserver-external-workload-certs"
+	ClusterMeshServerSecretName           = "clustermesh-apiserver-server-cert"
+	ClusterMeshAdminSecretName            = "clustermesh-apiserver-admin-cert"
+	ClusterMeshClientSecretName           = "clustermesh-apiserver-client-cert"
+	ClusterMeshExternalWorkloadSecretName = "clustermesh-apiserver-external-workload-cert"
```

Comment:
If we are changing these variables, won't they all be considered deprecated by the `hasDeprecatedName` function?

Reply:
Yes. Isn't that ok? When creating secrets we want to use these names, but when we are getting them, we are trying both these and deprecated names.

Reply:
> but when we are getting them, we are trying both these and deprecated names.

where is this being done?

Reply:
In `func (k *K8sClusterMesh) getSecret` in [clustermesh/clustermesh.go](https://github.com/cilium/cilium-cli/pull/1056/files/e86e8eddd01316d621872593c5a9053f7cfca554#diff-e710c4b2f4bc90c43de100d5a5d8743af75cadd136f7a24157700391a66ec3b6), line 669 and 670.

Reply:
@carnerito that function is not using the deprecated names as far as I can see.

Reply:
@aanm There are two commits. Change in getSecret func is in the second commit. It's my mistake, all changes should be in a single commit.